website/content/docs/architecture/improving-consul-resilience.mdx

```diff
@@ -0,0 +1,139 @@
+---
+layout: docs
+page_title: Improving Consul Resilience

```

Comment:
Should it be "Consul Resilience" or "Consul Server Resilience"?

I felt like "Consul Resilience" was better because that's what matters to the user. It's more of an underlying implementation/architectural detail that Consul's resilience is a function of its set of server agents and their configuration.